cdc_rsync/params.cc

```diff
@@ -189,6 +190,11 @@ OptionResult HandleParameter(const std::string& key, const char* value,
     return OptionResult::kConsumedKey;
   }
 
+  if (key == "version") {
+    params->options.version = true;
```

Comment:
Nit: Add version to params_test. IIRC there is a test that tests all params, be sure to add it there.